neuralmonkey/processors/helpers.py

```diff
@@ -17,3 +17,14 @@ def untruecase(
             yield [sentence[0].capitalize()] + sentence[1:]
         else:
             yield []
+
+
+def pipeline(processors: List[Callable]) -> Callable:
```

Comment:
To je dobrej nápad, ale možná bych tam nechal oba způsoby. V tom INI file nejde zalamovat řády a napsat pak do jednoho seznamu víc vícenásobných preprocesorů, tak by se to mohlo dost znepřehlednit.